src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle() {
+    }
+
+    public Circle(double radius) {
+        this.typeOfFigure = TypeOfFigure.CIRCLE;
+        this.area = getCircleArea(radius);
+        this.radius = radius;
```

Comment:
Creating a figure, don't pass expressions in the constructor. Create separate variables and pass them on for better code readability. Also, consider which variables should be class-level, as the 'typeOfFigure' can be set directly in the constructor without a method call.